net/unbound/files/unbound.sh

```diff
@@ -1047,9 +1026,28 @@ unbound_uci() {
 
 ##############################################################################
 
+unbound_ismain() # true=0, false=1.
+{
+  if [ "$UNBOUND_N_RX_PORT" != "53" ] ; then
```

Comment:
If dnsmasq is not present or disabled, then Unbound will manage reslove.conf (exclusive). If dnsmasq is enabled but dnsmasq doesn't have 53 and Unbound does, then Unbound will manage resolve.conf (no conflict). Else Unbound will not. Three 'if-elif-else' branches.

The common person use case.
Friend 1: My ISP is hijacking my web browsing with advertising insertion.
Friend 2: Make sure you have a separate cable modem and router.
Friend 1: Oh yeah. I did that like you said when I moved in.
Friend 2: OpenWrt ... update to latest LEDE 17.1.2 ?
Friend 1: Yeah, to the first, but should I update?
Friend 2: Update and then use the web software installer for "Unbound" and then go to menu and disable "dnsmasq"
Friend 1: Then what happens?
Friend 2: Recursion ... DNSSEC ... bottom line your ISP is not your DNS and cannot insert advertising through DNS.
Friend 1: Any setup for that?
Friend 2: For what you care about it will work "out of the box."